src/DefaultProxyUnleash.php

```diff
@@ -54,4 +55,13 @@ public function getVariant(string $featureName, ?Context $context = null, ?Varia
 
         return $variant;
     }
+
+    /**
+     * @codeCoverageIgnore
+     */
```

Comment:
This should be removed and the method should be called in [CoverageOnlyTest](https://github.com/Unleash/unleash-client-php/blob/main/tests/CoverageOnlyTest.php). The reasoning is that the code gets changed by an automated tool, so every line should at least be called to check that the tool did not introduce any fatal breaks.

Reply:
Will do that, when we decide how to progress with the the Unleash interface

Reply:
Removed it